libctl/structure.cpp

```diff
@@ -1300,8 +1297,18 @@ static meep::susceptibility *make_multilevel_sus(const multilevel_atom *d) {
   for (int t = 0; t < d->transitions.num_items; ++t) {
     int i = d->transitions.items[t].from_level - minlev;
     int j = d->transitions.items[t].to_level - minlev;
-    Gamma[i*L+i] += d->transitions.items[t].transition_rate;
-    Gamma[j*L+i] -= d->transitions.items[t].transition_rate;
+    Gamma[i*L+j+1] -= d->transitions.items[t].transition_rate;
+    Gamma[(i-1)*L+j+1] += d->transitions.items[t].transition_rate;
```

Comment:
Why isn't `Gamma[i*L+j]` and `Gamma[j*L+i]` correct, i.e. the (i,j) and (j,i) entries of the Γ matrix?